paths_test.go

```diff
@@ -12,12 +12,17 @@ import (
 
 func TestPaths(t *testing.T) {
 	const k = "APPDATA"
+	const VersionAstilectron = DefaultVersionAstilectron
```

Comment:
These lines are useless if I'm not mistaken

Reply:
Left as test reference VersionAstilectron from when it was a global.

Reply:
Since the behavior has changed I'd rather remove these lines and use `DefaultVersionAstilectron` and `DefaultVersionElectron` below